IdentityService/PresentationLayer/Extentions/ServicesExtention.cs

```diff
@@ -0,0 +1,50 @@
+﻿using DataAccessLayer.Data;
+using DataAccessLayer.Models;
+using DataAccessLayer.Repositories;
+using Microsoft.EntityFrameworkCore;
+using BAL.Profiles;
+using BAL.Services;
+using Microsoft.AspNetCore.Server.IIS.Core;
+using System.Runtime.CompilerServices;
+
+namespace PresentationLayer.Extentions
+{
+    public static class ServicesExtention
+    {
+        public static IServiceCollection AddServices
+            (this IServiceCollection services, IConfiguration config)
+        {
+            services.AddScoped<IClientRepo, ClientRepo>();
```

Comment:
Create different extension methods for each action(like for configuring db, add services, add repositories and etc.)

Reply:
Ok